### ISSUE TITLE
Intellij doesn't refer to correct version in maven repository

### ISSUE THREAD

**Issue by lebrot** [jgitver/jgitver-maven-plugin#53]
**version**: 0.4.0
**usage context**: Intellij IDEA: 2016.1.4

**Problem description**:  I have defined a dependency of one of my modules (let's refer it as A) with _${project.version}_ in other module (let's refer it as B).
Module A is not sub module of module B but they both share same root module.

As long as module A is imported in my intellij then module B recognize it and I can import and use its class and maven build goes fine.

I case this module A isn't exist in intellij but his artifact with generated version is exist in repository, then I can't use it for coding in my intellij. I'm getting in the editor an error that class can't be resolved.
Note: if I'll build the module using intellij plugin then it will pass although the errors in the editor.

My guess is that in IDE editor the dependencies resolving still works as default and it will use the original version to resolve each dependency.

Bottom line, we must that all project modules are imported in the IDE to manage work without editor errors.

**Comment by McFoggy** [comment]
can you please provide a sample project that demonstrate the issue?

**Comment by McFoggy** [comment]
@lebrot I have created a sample reproducer app https://github.com/McFoggy/jgitver-samples

```
git clone https://github.com/McFoggy/jgitver-samples
cd jgitver-samples
git checkout intellij
cd intellij/multi
mvn clean install
```

Can you please describe using this sample app, what you are doing that does not work in IntelliJ.
My fear is that you use jgitver with a real dynamic version _(for example using version with commit distance or commit SHA1)_ in which case each time you commit you would need to `mvn install` to make the libraries available to your IDE if they are not all loaded in your IDE.

**Comment by jschneider** [comment]
Might this be related to that issue I have created?
https://youtrack.jetbrains.com/issue/IDEA-172467

Multiple projects loaded into IntelliJ?

**Comment by McFoggy** [comment]
as stated in #70 Intellij now has full support for jgitver.
reopen if needed.
